README.md

```diff
@@ -76,6 +88,21 @@ spec:
   - implementations are additionally encouraged to support names that:
     - are up to 255 characters in length
     - contain lowercase alphanumeric characters or `-`, `.`, `|`, `/`, `\`
+- **metadata.labels:** *map[string]string|string[]* - optional field `key` <> `value`
+  - the `key` segment is required and must contain at most 63 characters beginning and ending
+     with an alphanumeric character `[a-z0-9A-Z]` with dashes `-`, underscores `_`, dots `.`
+     and alphanumerics between.
+  - the `value` of `key` segment can be a string or an array of strings
```

Comment:
May not be the right place to ask this question, but are there any specific reasons for `value` to be a string or an array of strings?

This choice complicates implementation of the data model as the two are represented differently in YAML, making it difficult to have a data model of a singular value type without a level of additional parsing to convert a single string to an array of strings with a single entry.

I might have missed alternative ways to resolve it, would appreciate feedback if that's the case

Reply:
I think that I agree with @kenfinnigan on this one.  We might be able to accomplish the same thing by removing the unique constraint for the key, and allowing multiple of the same keys with different values

Reply:
Fair enough, I was thinking it could be used for listing lists (in a pretty way?)

Reply:
So: merge this PR and then somebody pushes another one to address this?

Reply:
Sure, sounds good

Reply:
Cool